subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -76,7 +76,12 @@
                             <li><a href="/userguide/installation.html">Installing Gradle</a></li>
                             <li><a href="https://guides.gradle.org/creating-new-gradle-builds/">Creating New Gradle Builds</a></li>
                             <li><a href="https://guides.gradle.org/creating-build-scans/">Creating Build Scans</a></li>
-                            <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#migrating-from" aria-expanded="false" aria-controls="migrating-from">Migrating from...</a>
```

Comment:
I'm not sure if migration is the right term here. "Upgrade" feels more like it.

Reply:
Agreed. I was planning to update the nav for the upgrade guide. Now that the new layout has been merged in, I think I'll work against that.